lib/ox/element.rb

```diff
@@ -150,7 +150,18 @@ def locate(path)
     # _raise_ [NoMethodError] if no match is found
     def method_missing(id, *args, &block)
       has_some = false
-      ids = id.to_s
+      ids = id.to_s.sub('_', ':') # allows easy API with namespaces
```

Comment:
Underscore is a valid character for attribute names. By replacing it names such as _foo are no longer supported. If the users wants a : at the start they can just use a :, right?